lib/plugins/aws/deploy/lib/uploadArtifacts.js

```diff
@@ -25,12 +25,29 @@ module.exports = {
       this.options.region);
   },
 
-  uploadZipFile() {
+  uploadFunctions() {
     this.serverless.cli.log('Uploading .zip file to S3...');
 
-    const body = fs.readFileSync(this.serverless.service.package.artifact);
+    if (this.serverless.service.package.individually) {
```

Comment:
I just checked and it looks like this part of the function isn't really covered from a test:

https://coveralls.io/builds/7732943/source?filename=lib%2Fplugins%2Faws%2Fdeploy%2Flib%2FuploadArtifacts.js#L31

Could you look into why this might be the case? Not sure if its simply not covered by the test or if there is some issue going on. I assume its simply a missing test where package.individually is set to true.

I'll also make sure that the code coverage is more visible for users